src/index.html

```diff
@@ -1,14 +1,41 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
+    <title>Snake</title>
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
+  </head>
+  <body>
+    <div class="container">
+      <div class="block block--1">
+        <div class="block__text">1</div>
+      </div>
```

Comment:
```suggestion
      <div class="block">1</div>
```
Внутренний div не нужен, и используй `:nth-child` вместо дополнительных классов